src/common/utility.cpp

```diff
@@ -52,7 +52,13 @@
 #if defined(Q_OS_WIN)
 #include "utility_win.cpp"
 #elif defined(Q_OS_MAC)
-#include "utility_mac.cpp"
+// We cannot include utility_mac.mm, so it will be compiled separately. Below are the forward declarations.
+namespace OCC {
+void setupFavLink_private(const QString &folder);
```

Comment:
I guess we can declare them for all in a private header?
Maybe we can also get rid of those cpp includes...

Reply:
done